package.json

```diff
@@ -93,12 +93,12 @@
   "license": "MIT",
   "devDependencies": {
     "aws-sdk": "^2.404.0",
-    "ts-node": "^8.6.1",
+    "node-sass": "^4.13.1",
```

Comment:
No need to change, I'm just surprised this got reordered like this, presumably something npm decided...